src/main/kotlin/apply/utils/RandomPasswordGenerator.kt

```diff
@@ -0,0 +1,5 @@
+package apply.utils
```

Comment:
`application` 패키지에 있으면 어떨까요?

```suggestion
package apply.application
```